File: app/src/main/java/ml/dvnlabs/animize/pager/passdata_arraylist.java
/*
 * Copyright (c) 2020.
 * Animize Devs
 * Copyright 2019 - 2020
 * Davin Alfarizky Putra Basudewa <dbasudewa@gmail.com>
 * This program used for watching anime without ads.
 *
 */

package ml.dvnlabs.animize.pager;

import java.util.ArrayList;

import ml.dvnlabs.animize.model.VideoPlayModel;

public interface passdata_arraylist {
    void onDataReceived(ArrayList<VideoPlayModel> data, String id);
}

File: app/src/main/java/ml/dvnlabs/animize/ui/pager/PassDataArrayList.kt
/*
 * Copyright (c) 2020.
 * Animize Devs
 * Copyright 2019 - 2020
 * Davin Alfarizky Putra Basudewa <dbasudewa@gmail.com>
 * This program used for watching anime without ads.
 *
 */
package ml.dvnlabs.animize.ui.pager

import ml.dvnlabs.animize.model.VideoPlayModel
import java.util.*

interface PassDataArrayList {
    fun onDataReceived(data: ArrayList<VideoPlayModel?>?, id: String?)
}